Script Runner addon (#229)
xetys/hetzner-kube: docs/script-runner-addon.md

# Script Runner Addon Guide

Script Runner lets you run a bash script on all your cluster nodes with one command. It uploads the bash script to each node then execute. The script will receive two arguments upon execution:

> 1) Node group, for example (master, worker, etcd)

Reviewer: Markdown suggest to use:

```suggestion

1. Node group, allowed values are `master`, `worker` and `etcd`
```

> 2) Cluster info JSON config, which is similar to the hetzner-kube config found in `~/.hetzner-kube/config.json`

Reviewer: Markdown suggest to use
```suggestion
1. Cluster info JSON config, which is similar to the `hetzner-kube` configuration that you can  found in `~/.hetzner-kube/config.json`
```

### Example

> hetzner-kube Command:

Reviewer: Please add a space between title and text and between text and code sample

```bash
hetzner-kube cluster addon install --name demo script-runner /path/to/script.sh  
```
Example script.sh:
```bash

> 

Reviewer: Bash script should start with:
```
#!/bin/bash
```
this should be omitted (than specify on the doc that should not be included) or is missing here (than add to the sample)?
